packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -130,14 +62,14 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       break;
     }
 
-    deletePromises.push(
-      remindersToDelete.map((reminder) =>
```

Comment:
Resulted in a two-dimensional array and this caused issues with Promise.allSettled not working correctly